### DISCUSSION TITLE
"Fetching of tools from GRASS-Addons (be patient)" does not work on the new Debian 12, Bookworm

### DISCUSSION THREAD

**Discussion by sainokawara** [OSGeo/grass#3059] [Q&A]
I installed the brand new Debian 12, Bookworm, and the associated GRASS GIS 8 but when I attempted to the "Fetching of tools from GRAMSS-Addons", it never completes.  Actually, it probably never even started.  There is no Ethernet nor HDD activity, just the "spinning cursor" and that message at the bottom-left of the screen "Fetching of tools from GRASS-Addons (be patient)".  How patient do you have to be?  Oh, from the little I have used GRASS GIS 8, it is so much easier to use than 7!  Thank you, Harrison

**Comment by neteler** [comment]
> I installed the brand new Debian 12, Bookworm, and the associated GRASS GIS 8

Which version is it precisely? `g.version -rge` will tell (or the respective wxGUI entry).

> How patient do you have to be?

It should respond in seconds.
Are you behind a proxy?

> Oh, from the little I have used GRASS GIS 8, it is so much easier to use than 7! Thank you, Harrison

Yes, it is! :-)

**Reply by sainokawara** [reply]
Apparently, my "reply" of yesterday got lost.  They have installed GRASS GIS 8.2.1 on Debian 12.  And, no, there is no "proxy" in my 500 Mbps connection.  I hope this problem can be resolved without the need for a Wireshark trace!

**Comment by baswein** [comment]
I am having the same problem on windows with Grass 8.3.0 installed through OSGEO4W. Spinning circle and "please be patient fetching" and a "error: can't find executatble git" in the console.
